site/src/analytics-os-boards.ts

```diff
@@ -0,0 +1,119 @@
+import "@google-web-components/google-chart";
+import { GoogleChart } from "@google-web-components/google-chart";
+import {
+  css,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  query,
+} from "lit-element";
+import { AnalyticsDataCurrent } from "../../worker/src/data";
+
+const friendlyBoardName: Record<string, string> = {
+  "intel-nuc": "Intel NUC",
+  ova: "OVA",
+  "odroid-c2": "Odroid C2",
+  "odroid-c4": "Odroid C4",
+  "odroid-n2": "Odroid N2",
+  "odroid-xu4": "Odroid XU4",
+  rpi: "Raspberry Pi",
+  rpi0: "Raspberry Pi 0",
+  rpi2: "Raspberry Pi 2",
+  rpi3: "Raspberry Pi 3",
+  rpi4: "Raspberry Pi 4",
+  tinker: "Asus TinkerBoard",
+};
```

Comment:
@agners Is this list correct?

the board type (left) is the same as  `ha os info` outputs

Reply:
`generic-x86-64` is missing (friendly name `Generic x86-64`.

Some nits: I suggest to use all caps for `ODROID` (that is how they write it nowadays on their website, I guess we should transition to it).

ASUS is usually all caps as well.

Raspberry Pi 0 uses usually a written `Zero`.